src/content/docs/opentelemetry/get-started/collector-infra-monitoring/opentelemetry-collector-infra-intro.mdx

```diff
@@ -33,23 +33,23 @@ redirects:
   - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-statsd/
 ---
 
-The [OpenTelemetry collector](https://opentelemetry.io/docs/collector/) is a vendor-agnostic tool for receiving, processing, and exporting telemetry data. While collector requirements and configuration will vary, it comes with a variety of receivers and processors which make it popular for infrastructure monitoring. It's also common to use the collector for data processing, but this documentation focuses on infrastructure monitoring use cases. See [OpenTelemetry Collector for data processing](/docs/opentelemetry/get-started/collector-processing/opentelemetry-collector-processing-intro) for more information.
+The [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) is a vendor-agnostic tool for receiving, processing, and exporting telemetry data. While collector requirements and configuration will vary, it comes with a variety of receivers and processors which make it popular for infrastructure monitoring. It's also common to use the Collector for data processing, but this documentation focuses on infrastructure monitoring use cases. See [OpenTelemetry Collector for data processing](/docs/opentelemetry/get-started/collector-processing/opentelemetry-collector-processing-intro) for more information.
```

Comment:
`While collector requirements and configuration...`, collector is lowercase here.